### ISSUE TITLE
Should the title for a part answer input include its type?

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#834]
I think it would be helpful to include a part's type in the title for its input box, for example, "Answer for part a, mathematical expression".

It would be really nice to be able to show this information visually too. Maybe this information should be in the input hint, and it should always be displayed. For a mathematical expression part, it would be something like 'Write a mathematical expression'. That might be misleading for some contexts, though. 

This relates to #490

**Comment by christianp** [comment]
I think that this would duplicate information given in the hint display requested by #490, so we should just implement that. The hint would be screen-readable, so no need to add to the title.
